stdlib/public/SwiftOnoneSupport/SwiftOnoneSupport.swift

```diff
@@ -13,6 +13,8 @@
 //===----------------------------------------------------------------------===//
 import Swift
 
+@_fixed_layout // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
```

Comment:
this struct isn't used for anything right? I don't think you need to annotate it here